cds/modules/deposit/api.py

```diff
@@ -75,3 +78,117 @@ def _process_files(self, record_id, data):
     def multipart_files(self):
         """Get all multipart files."""
         return MultipartObject.query_by_bucket(self.files.bucket)
+
+
+def video_resolver(ids):
```

Comment:
@omelkonian  (if you was referring to this [test](https://github.com/CERNDocumentServer/cds/pull/254/files#diff-1c699497e5aaa8ed7b979398323dc496R148), I fixed, thanks :) )